src/app/core/services/asset-service-mapped.ts

```diff
@@ -25,7 +25,7 @@ export class AssetServiceMapped {
         assetPage.assets.map((asset) =>
```

Comment:
this parameter could be uiAsset, then you could abbreviate the `uiAsset: asset` to `uiAsset`